maven-core/src/main/java/org/apache/maven/lifecycle/DefaultLifecycles.java

```diff
@@ -43,7 +43,7 @@
 @Component( role = DefaultLifecycles.class )
 public class DefaultLifecycles
 {
-    public static final String[] STANDARD_LIFECYCLES = { "default", "clean", "site" };
+    public static final String[] STANDARD_LIFECYCLES = { "clean", "default", "site" };
```

Comment:
Does this change have any implications?

Reply:
I don't think so.  The lifecycles ordering is not much used, and the location where is was used was conflicting with the definition, hence the homogenisation of the two orders to make it more consistent.